Acrolinx.Sidebar/Util/Message/Message.cs

```diff
@@ -0,0 +1,18 @@
+﻿/* Copyright (c) 2021 Acrolinx GmbH */

```

Comment:
@mscc-sascha @mscc-sascha Use appropriate copyright header for new files as per guideline https://github.com/acrolinx/acrolinx-coding-guidance/blob/master/topics/formatting.md